samples/Calculator/README.md

```diff
@@ -0,0 +1,5 @@
+**Scenarios**
+
+1. [***Scenario 01*** Default settings (binary .coverage report)](scenarios/scenario01/README.md)
+2. [***Scenario 02*** Cobertura format by command line](scenarios/scenario02/README.md)
```

Comment:
that's fine for now...maybe in future we need to organize the index per "context/area" but fine for now imo.